projects/origin-release/golang-1.14/Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN yum install -y /tmp/origin-rpm/*.rpm && \
         gotest.tools/gotestsum \
         github.com/openshift/release/tools/gotest2junit \
         github.com/openshift/imagebuilder/cmd/imagebuilder && \
+    GO111MODULE=on go get gotest.tools/gotestsum@v0.5.2 && \
```

Comment:
Why doesn't this fit into the `go get` above?

Reply:
Because that doesn't have `GO11MODULE=on` set